File: Patterns/Symbols/25/symbol25.java
import java.util.*;
class symbol25{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number of lines: ");
        int n = sc.nextInt();
        sc.close();
        for (int i = 0; i < n; i++){
            for(int j = 0; j < n; j++){
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: Patterns/Symbols/25/symbol25.c
#include <stdio.h>
void main(){
    int n = 0,i,j;
    printf("Enter number of lines: ");
    scanf("%d",&n);
    for (i=0;i<n;i++){
        for (j=0;j<n;j++){
            printf("*");
        }
        printf("\n");
    }
}